src/tagmanager/tm_tag.c

```diff
@@ -82,7 +82,11 @@ typedef struct
 	gboolean first;
 } TMSortOptions;
 
-/* Gets the GType for a TMTag */
+/** Gets the GBoxed-derived GType for a TMTag.
```

Comment:
Should we advertise it's derived from GBoxed?  C code should probably have access to `tm_tag_ref()`/`tm_tag_unref()` instead as it has more meaningful semantics as `g_boxed_copy()`, and generic code can figure it out by itself looking at the type information.
And advertising GBoxed means it's kind of stuck to that implementation -- which is likely to say indeed, so I'm probably nitpicking, but we could want to change that one day.

Anyway, that's not a so important point.